src/AudioUnit/AudioUnit.cs

```diff
@@ -1954,7 +2011,8 @@ public enum AUSpatializationAlgorithm : uint
 		Hrtf = 2,
 		SoundField = 3,
 		VectorBasedPanning = 4,
-		StereoPassThrough = 5
+		StereoPassThrough = 5,
+		HRTFHQ = 6,
```

Comment:
Likely Hight Quality, so `HrtfHQ` (value of `2` + `HQ`) or `Hrtfhq`, would be correct .net names.

> And we now have a new rendering algorithm in this Spatial Mixer, called HRTFHQ, high quality.

See transcript of https://developer.apple.com/videos/play/wwdc2017/501/